assets/src/components/detours/activeDetourPanel.tsx

```diff
@@ -36,17 +40,19 @@ export const ActiveDetourPanel = ({
 
     <Panel.Body className="d-flex flex-column">
       <Panel.Body.ScrollArea>
-        {onNavigateBack && (
-          <Button
-            className="icon-link my-3"
-            variant="outline-primary"
-            onClick={onNavigateBack}
-          >
-            <ArrowLeft />
-            Back
-          </Button>
-        )}
-
+        <div className="d-flex flex-rown justify-content-between align-items-center">
+          {onNavigateBack && (
+            <Button
+              className="icon-link my-3"
+              variant="outline-primary"
+              onClick={onNavigateBack}
+            >
+              <ArrowLeft />
+              Back
+            </Button>
+          )}
+          <ExclamationTriangleFill className="c-active-detour__alert-icon" />
```

Comment:
```suggestion
          <ExclamationTriangleFill className="fs-2" />
```

Reply:
This replaces `width` and `height` css with `font-size` css, which doesn't achieve the same effect

Reply:
From what I understand and see on bootstraps docs, this _should_ work, so I'd expect something about our setup is broken if this _doesn't_ work (and I confirmed locally it doesn't)

Reply:
Nevermind, sorry!

Reply:
If you go to a bootstrap icon, and edit the classes defined in the HTML of an icon in the Examples section to have a `fs-<x>` you'll find that it does react to `fs-<x>`.

This works because they have `width` and `height` set to `1em`. I kindof like this usage and would consider making it a utility class to make the `icon-font-square` a reusable class since our icon sizes seem to lineup with font sizes. 

Just an idea for more reusable css but I'm good with this as is I think.